### ISSUE TITLE
Single Dice Roller Util adds the bonus to a damage roll multiple times if rolling more than one die

### ISSUE THREAD

**Issue by LichMaster98** [742617000027/farverona-sectormap#3]
> I don't know who makes the interactive map that adam was using, but i was messing with it and the crux grav tanks somehow rolled 26 for damage.  That should not be possible on 2d10+4 . I don't know how to else to report a bug. So just leaving this here if anyone can somehow let the guy know i guess lol 

From: [Discord Server](https://discordapp.com/channels/434794801058152468/535534804867743754/692942111955484672)

**Comment by LichMaster98** [comment]
I did verify that the script is correctly parsing the number of dice to roll, the face of the die, and the bonus of the roll.  Before I explain the bug, let me be clear that this only affects damage or counter dice which include more than one die roll AND a bonus to the attack roll, i.e. `2d10+4` (this is why the GravTank displays the bug). The code below is what is wrong. In rolling more than one attack die the bonus to the attack roll is added each time.  It's a simple mistake to fix in the code (and unfortunately errors are common in programming :frowning: ) To be clear, I can't find anything that would have impacted this turn given what we rolled, but yeah.
Impacted locations: https://github.com/742617000027/farverona-sectormap/blob/master/scripts/utils.js#L1302-L1304
 & https://github.com/742617000027/farverona-sectormap/blob/master/scripts/utils.js#L1361-L1363

There may be more locations but these are what I immediately found
